### DISCUSSION TITLE
4:1 ratio

### DISCUSSION THREAD

**Discussion by tomkettle1** [CASTChristmasForest2020/ForestSim#7] [Q&A] [answered]
@PotatoHax101 and I were talking about the tree planting algorithm & didn't agree on whether it is a random, 4:1 chance that the tree's are fir or spruce, or if it is an exact 4:1 ratio, and planted as 4 fir, 1 spruce, repeat. The wording is "10000 trees of which there is a 4:1 ratio of fir to spruce". Wanted others input

**Comment by TimMcGilly** [comment, marked as answer]
That is a good point. I was thinking random would be nice (and at first though would work) but if we are matching spec exactly we should make it more exact so for example every 5th tree is spruce. It will become more random looking pretty quickly with harvesting and fires and stuff. 

We will also need a different planting algorithm for stuff like replacement and forest fires. Not sure if this should be random or deterministic.

**Comment by tomkettle1** [comment]
Ok, all decided it probably doesn't matter so I'll close this
